site/content/docs/5.2/forms/input-group.md

```diff
@@ -64,6 +71,12 @@ Add the relative form sizing classes to the `.input-group` itself and contents w
 
 **Sizing on the individual input group elements isn't supported.**
 
+{{< ods-incompatibility-alert >}}
+This form variant, with an **horizontal layout** (i.e. labels not above the input fields), should not be used because it does not respect the Orange Design System specifications.
```

Comment:
Must be clearer IMO since the topic is not horizontal layout here but the sizing.

Reply:
What about:
> These 3 form variants, with an **horizontal layout** (i.e. labels not above the input fields), and the **small** one with a height of 28px, should not be used because they do not respect the Orange Design System specifications.